action/action.go

```diff
@@ -365,8 +365,8 @@ func getImageMap(b bundle.Bundle) ([]byte, error) {
 	return json.Marshal(imgs)
 }
 
-func opFromClaim(stateless bool, c claim.Claim, ii bundle.InvocationImage, creds credentials.Set) (*driver.Operation, error) {
-	env, files, err := creds.Expand(c.Bundle, stateless)
+func opFromClaim(stateless bool, c claim.Claim, ii bundle.InvocationImage, creds valuesource.Set) (*driver.Operation, error) {
+	env, files, err := expandCredentials(&c.Bundle, creds, stateless)
```

Comment:
As a suggested simplification, now that claim.Bundle is a value, we can change the signature of `expandCredentials` to take a value too and not need to change it to a pointer here and elsewhere (like in the tests) anymore.

Reply:
Good call; updated!